package.json

```diff
@@ -119,6 +119,23 @@
           "default": "",
           "description": "The path to your elm executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
         },
+        "elmLS.elmReviewPath": {
+          "scope": "window",
+          "type": "string",
+          "default": "",
+          "description": "The path to your elm-review executable. Should be empty by default, in that case it will assume the name and try to first get it from a local npm installation or a global one. If you set it manually it will not try to load from the npm folder."
+        },
+        "elmLS.elmReviewDiagnostics": {
+          "scope": "window",
+          "type": "string",
+          "enum": [
+            "off",
+            "warning",
+            "error"
+          ],
+          "default": "warning",
```

Comment:
Need to change this to `off`

Reply:
Maybe call it: `Enable experimental elm-review diagnostics.`

Reply:
@Razzeee Why do we want the default to be off?

Reply:
@jmbockhorst the response time is pretty long, especially on big codebases. Even though I still find this useful, maybe it's better to get the feedback first, then see how this could be improved by e.g. using the `watch` mode.

Reply:
Sounds good to me.

Reply:
It might also be confusing in some circumstances, where elm-review and elm-ls diagnostics shadow each other. Especially if you are not the person that setup the project.